Detectors/TOF/calibration/include/TOFCalibration/TOFDCSProcessor.h

```diff
@@ -115,8 +115,17 @@ class TOFDCSProcessor
     mTOFDCS.clear();
   }
 
- private:
-  std::unordered_map<DPID, TOFDCSinfo> mTOFDCS;                // this is the object that will go to the CCDB
+  bool areAllDPsFilled()
+  {
+    if (!it.second) {
+      return false;
```

Comment:
what is `it`? Was something cut out?

Reply:
This was my typo when removing some logs, I messed up the parenthesis. Sorry.